angular/src/app/discover/discover.component.ts

```diff
@@ -20,19 +22,33 @@ export class DiscoverComponent implements OnInit {
       this.getProjects();
     }
 
-    getProjects() {
-      this.discoverService.getProjects().subscribe(
-          (response: Array<any>) => {
-          this.projects = response;
-          },
-          error => {
-          this.error = true;
-          }
-      );
-    }
+  getProjects() {
+    console.log(this.user_id);
+    this.discoverService.getProjects().subscribe(
+      (response: Array<any>) => {
+        this.projects = response;
+      },
+      error => {
+        this.error = true;
+      }
+    );
+  }
 
-    onEdit(project) {
-      localStorage.setItem("project", JSON.stringify(project));
-      this.router.navigate(['/create']);
-    }
+  onEdit(project) {
+    localStorage.setItem("project", JSON.stringify(project));
+    this.router.navigate(['/create']);
+  }
+
+  onSearch(searchProjects: string[]) {
+    log(searchProjects);
```

Comment:
is `log` the same as console log? if it's just used for print debugging, please remove.

Reply:
Got it removed all non-error/debug. logging